test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class VisitorCanAddItemToCartAndViewCartTest < ActionDispatch::IntegrationTest
+  test 'visitor can add item to cart' do
+    skip
+    visit items_path
+
+  end
+  test 'visitor can view cart with selected items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.